sqldelight-compiler/integration-tests/src/test/kotlin/com/example/testmodule/TestDatabaseImpl.kt

```diff
@@ -32,11 +34,12 @@ private class TestDatabaseImpl(
 
   public override val teamQueries: TeamQueries = TeamQueries(driver, teamAdapter)
 
-  public object Schema : SqlDriver.Schema {
+  public object Schema : SqlDriver.Schema<SqlPreparedStatement, SqlCursor> {
     public override val version: Int
       get() = 1
 
-    public override fun create(driver: SqlDriver): Unit {
+    public override fun create(driver: SqlDriver<SqlPreparedStatement, SqlCursor>): Unit {
+      driver.execute(null, "CREATE TABLE `group` (`index` INTEGER PRIMARY KEY NOT NULL)", 0)
```

Comment:
Do you have any idea why the statements on line 42 and 60 have shifted? Does that matter?